packages/@uppy/aws-s3-multipart/src/index.js

```diff
@@ -22,6 +22,15 @@ function throwIfAborted (signal) {
   if (signal?.aborted) { throw createAbortError('The operation was aborted', { cause: signal.reason }) }
 }
 
+function abortable (promise, signal) {
+  const abortPromise = () => promise.abort(signal.reason)
+  signal.addEventListener('abort', abortPromise, { once:true })
+  const removeAbortListener = () => { signal.removeEventListener('abort', abortPromise) }
+  promise.then(removeAbortListener, removeAbortListener)
```

Comment:
`removeAbortListener` twice in the then?

Reply:
Yes, we want to remove the listener as soon as the promise settles (rejects or resolves).